src/worker/utils/chem.ts

```diff
@@ -215,6 +217,27 @@ export const addHs = (structure: string) => {
   }
 };
 
+export const getStereoTags = (structure: string) => {
+  const mol = get_molecule(structure, globalThis.workerRDKit);
+  if (!mol) throw new Error('@iktos-oss/rdkit-provider: mol is null');
+
+  try {
+    const stereoTags = mol.get_stereo_tags();
+    const { CIP_atoms: CIPAtoms, CIP_bonds: CIPBonds }: { CIP_atoms: CIPAtoms; CIP_bonds: CIPBonds } =
+      JSON.parse(stereoTags);
+
+    return {
+      CIPAtoms,
+      CIPBonds,
+    };
```

Comment:
can we avoid using the same name for the variable and the type? in `CIPAtoms` and `CIPBonds`

Reply:
Changed variables to `CIP_atoms` and `CIP_bonds`, same as what `get_stereo_tags()` returns.